src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -129,6 +129,20 @@ object DockerBuildPlugin extends AutoPlugin {
     )
 
     ////////////////////////////////////////////////////////////////////////////////////////////////
+    // The following settings affect how the plugin behaves, but don't affect file or image output.
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+
+    val verifyDockerfileIsStrict: SettingKey[Boolean] = Def.settingKey[Boolean](
+      "If true, the `verifyDockerfile` task will raise an error if the current Dockerfile does " +
+        "not match the generated one. Defaults to true."
+    )
+
+    val verifyDockerfileOnBuild: SettingKey[Boolean] = Def.settingKey[Boolean](
+      "If true, `dockerBuild` will depend on `verifyDockerfile`, and will not build if the " +
+        "Dockerfile is not up-to-date. Defaults to false."
```

Comment:
That makes sense. I think the warning would be a nice-to-have, but I wouldn't block this on it.  👍